assets/js/components/sell-quick-start.component.js

```diff
@@ -29,35 +29,45 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
   $scope.sellExchangeRate = {};
   $scope.changeSellCurrency = this.changeCurrency;
   $scope.tradingDisabled = this.tradingDisabled;
-  $scope.tradingDisabledReason = this.tradingDisabledReason;
   $scope.currencies = currency.coinifySellCurrencies;
   $scope.error = {};
   $scope.status = { ready: true };
   $scope.totalBalance = Wallet.my.wallet.balanceActiveAccounts / 100000000;
   $scope.sellTransaction.btc = null;
   $scope.selectedCurrency = $scope.sellTransaction.currency.code;
+  $scope.format = currency.formatCurrencyForView;
 
   let exchange = buySell.getExchange();
   $scope.exchange = exchange && exchange.profile ? exchange : {profile: {}};
   $scope.exchangeCountry = exchange._profile._country || $stateParams.countryCode;
   if ($scope.exchange._profile) {
     $scope.sellLimit = $scope.exchange._profile._currentLimits._bank._outRemaining.toString();
+    $scope.hideIncreaseLimit = $scope.exchange._profile._level._name > 1;
   }
 
-  const setInitialCurrencyAndSymbol = (code, name) => {
-    $scope.sellTransaction.currency = { code: code, name: name };
-    $scope.sellCurrencySymbol = currency.conversions[code];
-    $scope.limitsCurrencySymbol = currency.conversions[code];
-  };
-
-  if ($scope.exchangeCountry === 'DK') {
-    setInitialCurrencyAndSymbol('DKK', 'Danish Krone');
-  } else if ($scope.exchangeCountry === 'GB') {
-    setInitialCurrencyAndSymbol('GBP', 'Great British Pound');
-  } else {
-    setInitialCurrencyAndSymbol('EUR', 'Euro');
+  if (this.tradingDisabledReason === 'awaiting_first_trade_completion' && this.pendingTrade.medium === 'blockchain') {
+    $scope.isPendingSellTrade = true;
```

Comment:
This will only be set when the directive is rendered, so if a sell trade is completed while the user is there, it will still be set to `true`. Intentional? Quick fix would be to encapsulate this in a function `isPendingSellTrade -> Bool` (similar to `isPendingTradeState`).